src/drawing/DrawingTarget.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import * as ReactRedux from 'react-redux';
+import classNames from 'classnames';
+import noop from 'lodash/noop';
+import { getIsCurrentFileVersion } from '../store';
+import { MOUSE_PRIMARY } from '../constants';
+import './DrawingTarget.scss';
+import { PathGroup, Shape } from '../@types';
+
+type Props = {
+    annotationId: string;
+    className?: string;
+    isActive?: boolean;
+    onSelect?: (annotationId: string) => void;
+    pathGroups: PathGroup[];
+    shape: Shape;
+};
+
+export type DrawingTargetRef = HTMLAnchorElement;
+
+export const DrawingTarget = (props: Props, ref: React.Ref<DrawingTargetRef>): JSX.Element => {
+    const { annotationId, className, isActive = false, onSelect = noop, shape } = props;
+    const isCurrentFileVersion = ReactRedux.useSelector(getIsCurrentFileVersion);
+    const { height, width, x, y } = shape;
+
+    const handleFocus = (): void => {
+        onSelect(annotationId);
+    };
+    const handleMouseDown = (event: React.MouseEvent<DrawingTargetRef>): void => {
+        if (event.buttons !== MOUSE_PRIMARY) {
+            return;
+        }
+
+        event.preventDefault(); // Prevents focus from leaving the button immediately in some browsers
```

Comment:
There's also a workaround we had to do for IE11 that may apply here: https://github.com/box/box-annotations/blob/2a0e8c6483eb68d721ca40337180b803eacd6bfe/src/highlight/HighlightTarget.tsx#L60